libxdamage/plan.sh

```diff
@@ -0,0 +1,34 @@
+pkg_name=libxdamage
+pkg_origin=core
+pkg_version=1.1.4
+pkg_maintainer="The Habitat Maintainers <humans@habitat.sh>"
+pkg_description="X11 C Bindings"
+pkg_upstream_url="https://www.x.org/"
+pkg_license=('MIT')
+pkg_dirname="libXdamage-${pkg_version}"
+pkg_source="https://www.x.org/releases/individual/lib/${pkg_dirname}.tar.bz2"
+pkg_shasum="7c3fe7c657e83547f4822bfde30a90d84524efb56365448768409b77f05355ad"
+pkg_deps=(
+  core/glibc
+  core/libpthread-stubs # not linked by libs/bins
```

Comment:
`libpthread-stubs` only provides a pkg-config config which on linux (the only platform we rally support with these packages at the moment) does nothing.

You /should/ be able to move this to a build_dep, if not, I'm guessing the problem is some quirk of how we put together the PKG_CONFIG_PATH.

Reply:
Yeah I added the comment where I wasn't sure. I think you're right, I'll test it when I have time to ;)

Reply:
Done, thanks :)